### PR TITLE
Terminology changes and descriptions

### PR DESCRIPTION
**Pull request by justinegeffen**
Terminology project handbook page.

@cwarnermm: This is a working snapshot of the terminology changes and work. It'll be useful in situations where someone wants to know what the project entails and what we've done. 

The links are to the GDoc. The spreadsheet isn't quite there yet but will be added when it's in better shape.

As this is a WIP doc, I'd like to merge it and continue to iterate on it.

### CODE REVIEW

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+The language we use at Mattermost is extremely important. When we use consistent terminology it means we're all speaking the same language, and our customers know what to expect whether they're on the website or in the product documentation.
+
+Terminology standardization started in 2022. The project has a large scope and includes:
+
+- Standardizing headings to make them sentence case.
+- Standardizing all text in the System Console to make them sentence case.
+- Standardizing how we refer to our products and plans, as well as the mechanisms available to use them.
+- Changing the casing of words to lowercase where appropriate.
+- Making our Edition and License page consistent and clear.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]: May want to clarify that we're talking about the Edition and License page within the System Console

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- Creating a UI text library of terms and strings for the UX team to use when creating mocks. This can also be used cross-functionally and by external partners.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]:
```suggestion
- Creating a UI text library of terms and strings for teams and external partners to use when creating mockups, designs, wireframes, and integrations.
```

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+The [outline of the project is documented here](https://docs.google.com/document/d/1ImzS3UhCkFzMF2-6FNByw9W2r8Gvx1sudBA0FjazVH0/edit#heading=h.crjl4fz9pz1n).
+
+## Sentence case headings
+
+We decided to change our headings to sentence case to:
+
+* Decrease friction for contributors so they wouldn't have to "take a stab" at getting heading capitalization right.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]: "guess" might be another plain English term for the idiom "take a stab" :)

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+* Bring our documentation format in line with current industry standards around readability.
+* Make our documentation easier to read.
+
+This is a work in progress with the documentation having been completed in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.
```
Comment (cwarnermm) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]: are the docs complete? maybe safer to say "...with documentation updates delivered in Q1 of 2022."

**Review by cwarnermm — DISMISSED** [review]
Really great update! It's clear and informative! Thank you! I've provided feedback to consider inline. Approving for expediency.

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md (hunk 3 above)
Comment (justinegeffen) [reply operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]: That is very true, thank you!

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md (hunk 1 above)
Comment (justinegeffen) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]:
```suggestion
- Making our Edition and License page in the System Console consistent and clear.
```

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- Making our Edition and License page in the System Console consistent and clear.
+- Creating a UI text library of terms and strings for teams and external partners to use when creating mockups, designs, wireframes, and integrations.
+
+The [outline of the project is documented here](https://docs.google.com/document/d/1ImzS3UhCkFzMF2-6FNByw9W2r8Gvx1sudBA0FjazVH0/edit#heading=h.crjl4fz9pz1n).
+
+## Sentence case headings
+
+We decided to change our headings to sentence case to:
+
+* Decrease friction for contributors so they wouldn't have to "take a stab" at getting heading capitalization right.
```
Comment (justinegeffen) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]:
```suggestion
* Decrease friction for contributors so they wouldn't have to guess at getting heading capitalization right.
```

File: operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+* Bring our documentation format in line with current industry standards around readability.
+* Make our documentation easier to read.
+
+This is a work in progress with the documentation having been completed in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.
```
Comment (justinegeffen) [inline operations/research-and-development/product/technical-writing-team-handbook/our-terminology.md]:
```suggestion
This is a work in progress with documentation updates delivered in Q1 of 2022. The scope of the project extends to the Mattermost website, developer documentation, Handbook, and integrations content. There is no date set for completion.
```

**Pull request merged**